backend-config/account_merger_test.go

```diff
@@ -0,0 +1,242 @@
+package backendconfig
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMergeAccounts(t *testing.T) {
```

Comment:
the name of the test file should follow the name of the actual file that it tests, e.g. `account_association_test.go`